doc/arch/adr-007-pa-dep-client-app.md

```diff
@@ -0,0 +1,122 @@
+
+Advantages of having **different repositories**:
+
+  * Cleaner logical separation in the codebase
+  * Fewer cohabiting developer entry points, i.e. `package.json`, or multiple `node_modules`
+  * Does not require changing the `node` or `npm` version in MMW
+  * Does not require adding a Docker environment in MMW for building the front-end
+  * Faster deployment, as the client app can be deployed independently of the main app
+
+### Deployment: Paired vs Independent
+
+Advantages of **paired deployment**:
+
+  * Front-end and back-end will always be in sync
+  * Does not require extra configuration of AWS
+  * Does not need extra steps during deployment
+  * Easier to write end-to-end tests
+
+Advantages of **independent deployment**:
+
+  * Front-end can be iterated much faster, without the heavy deployment cycle of MMW
+
+### Access: Subdomain vs Subdirectory
+
+Advantages of **subdomain access**:
+
+  * More easily supports independent deployments
+
+Advantages of **subdirectory access**:
+
+  * Maintains MMW brand identity
+  * Would not have any CORS issues
+    - This would not apply to the documented public API endpoints,
+      but there are some undocumented APIs such as MapShed which we'll need to run,
+      which may have CORS headers set on them.
+
+### Option A: Same Repository, Paired Deployment, Subdirectory Access
+
+Similar to Beekeepers,
+we will create a Django app to house all files for the new client app.
+We will evaluate if the `node` version in the App VM can be upgraded,
+and if not we will use add a Docker container for building the front-end.
+Django will be configured to handle routing,
+which will render a different home page for the new routes.
+Deployments will not be affected at all and will proceed as usual.
+Some basic styles and colors will be shared between the two apps.
+
+### Option B: Different Repositories, Independent Deployments, Subdirectory Access
+
+The new app gets its own repository,
+which makes the code much easier to navigate.
+During development, the MMW VM would have to be running in addition to the client app.
+The app will be deployed to a static host like S3 or Netlify.
+nginx will be configured to proxy its contents at the subdirectory route.
+Deployments of cosmetic changes to the new client app can be made independently,
+but those of functional changes to how the API is consumed will have to be coordinated with MMW.
+Project management will be split across the two repos,
+with front-end tasks listed on the new repo and back-end tasks listed on MMW.
+
+## Decision
+
+The main argument for having a separate repository is better code separation,
+and faster, leaner deployment cycles.
+This is overshadowed by the simpler deployments and project management,
+easier ability to keep the front- and back-ends in sync,
+and ability to write end-to-end tests,
+if we keep the same repository.
+This also implies paired deployments and subdirectory access.
```

Comment:
Related to [Kelly's comment above](https://github.com/WikiWatershed/model-my-watershed/pull/3170/files#r328138808) consider adding a some explanation of why it does not make sense to have a single repository and deploy the new app to a subdomain.